controllers/controller.js

```diff
@@ -28,9 +30,27 @@ const findSpecificArticle = (request, response, next) => {
     .catch(next);
 };
 
+const findArticleComments = (request, response, next) => {
+  if (isNaN(Number(request.params.article_id))) {
+    response.status(400).send({ msg: "400 - Bad request" });
+  }
+  findHighestArticleId().then((highestArticle_Id) => {
+    if (highestArticle_Id[0].article_id < request.params.article_id) {
+      response.status(404).send({ msg: "404 - Not found" });
```

Comment:
The error handling above has to be done in the model as that's where we'd ideally be throwing error from. And instead of sending error responses in the model we want to throw promise rejections that can be caught in the error handling middleware